packages/web/src/env.mjs

```diff
@@ -21,6 +21,7 @@ export const env = createEnv({
         
         // Auth
         FORCE_ENABLE_ANONYMOUS_ACCESS: booleanSchema.default('false'),
+        FORCE_MEMBER_APPROVAL_REQUIRED: booleanSchema.optional(),
```

Comment:
following the rest of the changes to only force UI stuff when this is enabled, we can also default this to false to match the other env var